pkgs/applications/misc/blender/default.nix

```diff
@@ -385,7 +385,8 @@ stdenv.mkDerivation (finalAttrs: {
     license = with lib.licenses; [ gpl2Plus ] ++ lib.optional cudaSupport unfree;
     platforms = [
       "aarch64-linux"
-      "x86_64-darwin"
+      # the current apple sdk is too old (currently 11_0) and fails to build "metal" on x86_64-darwin
+      # "x86_64-darwin"
```

Comment:
I'd suggest keeping it commented under platforms. This way it will still spawn an error when trying to build on a `x86_64-darwin` machine and it isn't another attribute set we add (platforms vs platforms and broken). But I'm impartial to this ;-)

Reply:
It still should be in platforms, because upstream supports it there. We could have it marked as broken `meta.broken = stdenv.hostPlatform.system == "x86_64-darwin"`.